pages/account.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+import styles from "@/styles/pages/account.module.scss";
+import AccountUserName from "@/components/account/AccountUserName";
+
+export default function Account() {
+	const daysAlive = 9;
+	const [profilePicSrc, setProfilePicSrc] = useState("/lio_img.png");
+
+	const onUpload = (event) => {
+		try {
+			const file = event.target.files[0];
+			setProfilePicSrc(URL.createObjectURL(file));
+		} catch {}
+	};
+
+	return (
+		<main id={styles.main}>
+			<label for='profilePicInput' className={styles["profile-pic-input"]}>
+				<img src={profilePicSrc} alt="user's photo" />
+			</label>
+			<AccountUserName />
+			<input type='file' id='profilePicInput' onChange={onUpload} accept='image/png, image/jpg, image/jpeg' />
```

Comment:
the input should be inside of the label or at least be next to it below. The dividing Component makes things unclear.

Reply:
noted